src/endpoints/Billing/types.ts

```diff
@@ -0,0 +1,34 @@
+export enum SignupPlan {
+    STARTER = 'starter',
+    CORE = 'core',
+    PREMIUM = 'premium',
+}
+
+export enum SignupCurrencies {
+    EUR = 'eur',
+    USD = 'usd',
+}
+
+export enum SignupBillingCycles {
+    YEAR = 'year',
+    MONTH = 'month',
+}
```

Comment:
These are enum values, I wouldn't use plural, so: `SignupCurrency`, `SignupBillingCycle`. Maybe even without the `Signup` prefix.

Reply:
We already have such enums like here https://github.com/prezly/javascript-sdk/blob/master/src/types/Plan.ts
but looks like they do not much with this code https://github.com/prezly/prezly/blob/86312d46d76157f2b5d6a9714a13423522af1f83/lib/Prezly/Support/Plans/Plan.php#L28

can i use enums from shared types, or i have to create my own for signup?

Reply:
hm, hence i have to create my own enums, thanks)